Rakefile

```diff
@@ -75,7 +75,14 @@ task confirm_documentation: :generate_cops_documentation do
   end
 end
 
-task default: %i[build_config spec
+desc 'Run RSpec with Prism'
+task :prism_spec do
+  sh('PARSER_ENGINE=parser_prism bundle exec rake spec')
+end
+
+task default: %i[build_config
+                 spec
+                 prism_spec
```

Comment:
This would be just a duplicate rspec run for all jobs except the prism one, won’t it?

I suggest to either:
 - run all tasks with that env var, including internal investigations, doc gen etc
 - skip prism_spec from default - it’s run as a step of a prism job

Reply:
It was in line with rubocop-performance and rubocop-rails, but it could certainly be removed from the rake default.
https://github.com/rubocop/rubocop-performance/pull/446

Reply:
This is where having a second pair of eyes helps.